.husky/commit-msg

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+. "$(dirname -- "$0")/_/husky.sh"
+
+# The commit message, passed as an argument to this script
+# commit_message=$(cat "$1")
+commit_message_file=$(cat "$1")
+
+# if [[ "$commit_message" == '' ]];then
+#     echo "msg is empty"
+#     exit 1
+# fi
+
+# Define the regex for the Conventional Commit format
+
+commit_regex='^(build|ci|docs|feat|fix|perf|refactor|style|test|chore)(\(.+\))?: .{1,50}(\n\n.+)?'
+
```

Comment:
This will only check the first line

Is there a way to also check 
1. if ticket id is added
2. Body is written with * etc?

Reply:
will check and update